open-source-hosts-plugin/scripts/AwsTools/SumerianHostScript.ts

```diff
@@ -0,0 +1,207 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {HostObject} from '@amazon-sumerian-hosts/babylon';
+import {Mesh, Observable} from '@babylonjs/core';
+import * as AWS from 'aws-sdk';
+import {fromScene, visibleInInspector} from '../decorators';
+import IAwsConnector from './IAwsConnector';
+
+/**
+ * This is the script attached to a Sumerian Host. You will typically not need
+ * to edit this script. Instead, other scripts in your scene should interact 
+ * with the host via the `SumerianHost.host` property which provides a
+ * HostObject instance.
+ * 
+ * Host initialization is asynchronous. Your scripts can subscribe to the
+ * SumerianHost.onHostReadyObserver to be notified when the host is fully
+ * initialized and ready for use.
+ * 
+ * Example:
+ * _In your own script..._
+ ```
+// Assume hostNode references a SumerianHost instance.
+hostNode.onHostReadyObserver.add(() -> {
+  hostNode.host.TextToSpeechFeature.play("Hello, world!");
+});
+ ```
+ */
+
+type SumerianHostMetadata = {
+  bindPoseOffsetName: string;
+  poiConfigPath: string;
+  gestureConfigPath: string;
+  animClipPaths: any;
+  lookJoint: string;
+};
+
+// Our Github Actions will replace this with a commit SHA at release time
+// Right now this script is the only runtime asset published by our plugin
+// As we build out more runtime complexity into this plugin, this versioning should move there
+export const PLUGIN_VERSION = 'development';
```

Comment:
This file path needs to be updated here as well https://github.com/aws-samples/aws-tools-for-babylonjs-editor/blob/main/.github/workflows/host-plugin-pull-request.yml#L23

Reply:
I will fix.

Reply:
This is now fixed in the latest PR update. It seems like there's high risk for regression here. Any ideas on how we might be able to implement an automated test to confirm that the proper plugin version makes it through to Polly API calls?

Reply:
Yes agreed there is risk here. idea: The Action could be updated to fail if it doesn't do a replacement?